test/templates/pages/floorplans/details.njk

```diff
@@ -119,14 +117,6 @@
         {{ form_item("text", { label: "Store name" }) }}
       {%- endcall %}
     {%- endcall %}
-    {%- call layout.col(2) -%}
-      {% call form_set(properties={
-              legend: "Activate"
-          }) %}
-          {{  time_picker()
-          }}
-      {%- endcall %}
-    {%- endcall %}
```

Comment:
Lots of cleanup of time-picker AND date-picker in some of these other pages (that cause the playground to break since the templates were removed).

Reply:
Woops 🙍  Thanks!